NEWS.rst

```diff
@@ -0,0 +1,20 @@
+History
+=======
+
+0.6 (2016-12-05)
+----------------
+
+* Fix pip installation, clean up setup procedure, flake8 and add README
+* PiecewisePolynomial1DFit updated to work with scipy 0.18.0
+* Delaunay2DScatterFit now based on scipy.interpolate.griddata
+* Delaunay2DGridFit removed as there is no equivalent anymore
+
```

Comment:
Add a note that the 'nn' interpolation is no longer available, and the default is now cubic.